nl_service_metadata_generator/data/templates/partials/inspire_conformance_sds.xml

```diff
@@ -0,0 +1,68 @@
+                        </gmd:title>
+                        <gmd:date>
+                            <gmd:CI_Date>
+                                <gmd:date>
+                                    <gco:Date>2010-12-08</gco:Date>
+                                </gmd:date>
+                                <gmd:dateType>
+                                    <gmd:CI_DateTypeCode codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#CI_DateTypeCode" codeListValue="publication"/>
+                                </gmd:dateType>
+                            </gmd:CI_Date>
+                        </gmd:date>
+                    </gmd:CI_Citation>
+                </gmd:specification>
+                <gmd:explanation>
+                    <gco:CharacterString>Conform verordening</gco:CharacterString>
+                </gmd:explanation>
+                <gmd:pass>
+                    <gco:Boolean>true</gco:Boolean>
+                </gmd:pass>
+            </gmd:DQ_ConformanceResult>
+        </gmd:result>
+    </gmd:DQ_DomainConsistency>
+</gmd:report>
+<gmd:report>
+    <gmd:DQ_DomainConsistency>
+        <gmd:result>
+            <gmd:DQ_ConformanceResult>
+                <gmd:specification>
+                    <gmd:CI_Citation>
+                        <gmd:title>
+                            <gmx:Anchor xlink:href="{{sds_category_uri}}">{{sds_category}}
+                            </gmx:Anchor>
+                        </gmd:title>
+                        <gmd:date>
+                            <gmd:CI_Date>
+                                <gmd:date>
+                                    <gco:Date>2014-12-11</gco:Date>
+                                </gmd:date>
+                                <gmd:dateType>
+                                    <gmd:CI_DateTypeCode codeList="http://www.isotc211.org/2005/resources/codeList.xml#CI_DateTypeCode" codeListValue="publication"/>
+                                </gmd:dateType>
+                            </gmd:CI_Date>
+                        </gmd:date>
+                    </gmd:CI_Citation>
+                </gmd:specification>
+                <gmd:explanation>
+                    <gco:CharacterString>De service voldoet aan de requirements van de {{sds_category}} conformance class</gco:CharacterString>
+                </gmd:explanation>
+                <gmd:pass>
+                    <gco:Boolean>true</gco:Boolean>
+                </gmd:pass>
+            </gmd:DQ_ConformanceResult>
+        </gmd:result>
+    </gmd:DQ_DomainConsistency>
+</gmd:report>
+{% include "partials/inspire_conformance_technical_spec.xml" %}
```

Comment:
Voor SDS services moet geen inspire technical spec worden opgenomen, Hier verwachten een conformance met de WFS 2.0 of WMS 1.3 spec. Zie bijvoorbeeld https://nationaalgeoregister.nl/geonetwork/srv/dut/catalog.search#/metadata/ee9e7a1c-b04a-45a9-92a2-99b2ef508529?tab=inspire